dynamicscrm/metadata_test.go

```diff
@@ -40,119 +42,96 @@ func TestListObjectMetadata(t *testing.T) { // nolint:funlen,gocognit,cyclop
 		},
 		{
 			name:  "Mime response header expected",
-			input: []string{"msfp_surveyinvite"},
+			input: []string{"accounts"},
 			server: httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 				w.WriteHeader(http.StatusTeapot)
 			})),
 			expectedErrs: []error{interpreter.ErrMissingContentType},
 		},
 		{
-			name:  "Missing XML response on status OK",
-			input: []string{"msfp_surveyinvite"},
+			name:  "Schema endpoint is not available for object",
```

Comment:
Can you mention in the metadata code how we currently handle objects that don't respond to schema endpoints?

Reply:
Added a comment at the top of `ListObjectMetadata` method.